src/app-layer-smtp.h

```diff
@@ -124,7 +124,7 @@ typedef struct SMTPState_ {
     /** current command in progress */
     uint8_t current_command;
     /** bdat chunk len */
-    uint32_t bdat_chunk_len;
+    uint64_t bdat_chunk_len;
```

Comment:
seems like we're going to be wasting space here. Didn't immediately see a limit for BDAT chunk len. Wireshark uses u32. Haven't checked others.

Reply:
Fixing by using `StringParseUint32` instead of `strtoul`